src/AlfangoDB/modules/read.mo

```diff
@@ -179,4 +180,29 @@ module {
         Prelude.unreachable();
     };
 
+    public func getDatabaseInfo(alfangoDB: Database.AlfangoDB) : OutputTypes.GetDatabasesOutputType {
```

Comment:
Lastly, let us change this function name to GetDatabases in order to make it with input and output type names.